pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -234,6 +234,12 @@ func TestDesiredExternalDNSDeployment(t *testing.T) {
 							"--fqdn-template={{.Name}}.test.com",
 							"--txt-prefix=external-dns-",
 						},
+						Env: []corev1.EnvVar{
+							{
+								Name:  "SSL_CERT_DIR",
+								Value: "/etc/pki/ca-trust/extracted/pem",
```

Comment:
Is this an existing directory on the image?

Reply:
If it's a RHEL distribution: yes. For the others: it depends but mostly no.
Btw this directory will be created if not there by the volumeMount.

Reply:
Yeah, the `volumeMount` creation was there before, it was implemented as part the support of the proxy's trustedCA: [code](https://github.com/openshift/external-dns-operator/blob/ccc4fbbc3a2965886dce8c961d8a9038c7f5176e/pkg/operator/controller/externaldns/pod.go#L223-L229).